cdk/lib/prism.ts

```diff
@@ -94,5 +101,7 @@ export class PrismStack extends GuStack {
         allowConnectionsFrom: [Peer.ipv4("10.0.0.0/8")],
       },
     });
+
+    appServerSecurityGroup.connections.allowFrom(loadBalancer, Port.tcp(9000), "Port 9000 LB to fleet");
```

Comment:
This is necessary to prevent downtime in the period _after_ the CloudFormation update but _before_ the new boxes have been added to the load balancer. For a discussion of similar issues, see https://github.com/guardian/cdk/discussions/212.

We'll be able to remove this in a subsequent PR.